src/ape/managers/config.py

```diff
@@ -19,7 +17,6 @@ class DeploymentConfig(ConfigItem):
     contract_type: str
 
 
-@dataclass
```

Comment:
We either need an `init` method or to make this a data class again because we are failing it initialize it

Reply:
Well, now everything around where the injected dependency is also a class var lol. They should be set as instance vars now